Engine.Core/TestHost.fs

```diff
@@ -1,11 +1,13 @@
 ﻿namespace R4nd0mApps.TddStud10.Engine.Core
-
+open System.IO
+open System.Runtime.Serialization.Json
+open System.Text
 
 module TestHost =
     open R4nd0mApps.TddStud10.Common.Domain
 
     let buildCommandLine cmd (rsp : RunStartParams) =
-        sprintf "\"%s\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\""
+        sprintf "\"%s\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\" \"%O\""

```

Comment:
Yes, I assume that users will put namespaces/class names here and will be limited. We can send it by a named pipe, but lets stick with this for now.